troposphere/static/js/components/admin/AtmosphereUser.react.js

```diff
@@ -0,0 +1,89 @@
+define(function (require) {
+  "use strict";
+
+  var React = require('react/addons'),
+    Backbone = require('backbone'),
+    moment = require('moment'),
+    Router = require('react-router'),
+    actions = require('actions'),
+    ToggleButton = require('components/common/ToggleButton.react'),
+    stores = require('stores');
+
+  return React.createClass({
```

Comment:
Duly noted, we should make this change wholesale as part of a larger pull request for code-cleanup and the like.